intermediate_source/dist_pipeline_parallel_tutorial.rst

```diff
@@ -0,0 +1,10 @@
+Distributed Pipeline Parallelism Using RPC
+==========================================
+
+This tutorial has been deprecated.
+
+Redirecting in 3 seconds...
+
+.. raw:: html
+
+   <meta http-equiv="Refresh" content="3; url='https://pytorch.org/tutorials/index.html'" />
```

Comment:
We can redirect users to the new pipeline tutorial. https://pytorch.org/tutorials/intermediate/pipelining_tutorial.html